azure app-service module - closes #199 (#804)
bob-crutchley/courseware: topics/azure/modules/app-service/README.md

# Azure App Service

<!--TOC_START-->
## Contents
- [Overview](#overview)
- [App Service](#app-service)
- [Why?](#why)
	- [Multiple languages and frameworks](#multiple-languages-and-frameworks)
	- [DevOps](#devops)
	- [Scalability](#scalability)
	- [Serverless code](#serverless-code)
	- [Security and Compliance](#security-and-compliance)
- [Tutorial](#tutorial)
- [Exercises](#exercises)

<!--TOC_END-->
## Overview

In this module we are going to introduce the Azure App Service.

## App Service

Azure's App Service hosts web applications, REST APIs and mobile back ends.

> These applications can be made in many languages, scale and run with ease.

Reviewer: ```suggestion
These applications can be made in many languages, are scalable and run with ease.
```

They support both Windows and Linux environments.
It allows you to deploy any of your apps, in the way you prefer to do so. 

## Why?

Why should we use Azure App Service?

### Multiple languages and frameworks

When using the App Service you have a wide range of options when it comes to the languages you use.
You can also run scripts as background services to run your app the way you see fit.

### DevOps

> You can set up `C`ontinuous `I`ntegration (`CI`) and `C`ontinuous `D`eployment (`CD`) with Azure DevOps, Github, BitBucket, Docker Hub and many other options.

Reviewer: ```suggestion
Your app can still benefit from `C`ontinuous `I`ntegration (`CI`) and `C`ontinuous `D`eployment (`CD`) with Azure DevOps, Github, BitBucket, Docker Hub and many other options.
```
